packages/react-sdk/src/components/Layout/useFullscreenMode.ts

```diff
@@ -64,6 +64,21 @@ export function useFullscreenMode(): UseFullScreenModeResult {
     };
   }, [setFullscreenModeState]);
 
+  useEffect(() => {
+    function listener(e: KeyboardEvent) {
```

Comment:
We use [React Hotkeys Hook](https://react-hotkeys-hook.vercel.app/) in other places. It should give us more control over hotkeys in different context (such as only enable them in specific places).

Can you switch to it?

Reply:
@weeman1337 I tried but it seems like either it works 2 times and then gets stuck in `TypeError: Fullscreen request denied` in firefox or changing to keyup makes it work once and then not trigger again. So something seems wonky with this hook which makes it unreliable compared to using a classic keyhandler

Reply:
The hook seems to cause mainly `Anfrage für Vollbildmodus wurde abgelehnt, weil Element.mozRequestFullScreen() nicht aus einem kurz laufenden Benutzer-generierten Ereignis-Handler aufgerufen wurde.`

Reply:
It seems like it works when I actively click on the board in between but it doesn't just with the keybind despite https://developer.mozilla.org/en-US/docs/Web/Security/User_activation claiming keydown is accepted. I can only assume that firefox fails due to some indirection or memo thing here

Reply:
Then lets keep it like it. Adding a comment

Reply:
Also getting this without the hotkeys hook… Seems like it is triggered twice.

Reply:
I think this can be caused by Reacts strict mode.

Reply:
Or the hook being used in two places.